lib/utilities/collectResponseHeaders.js

```diff
@@ -77,6 +92,17 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
             objectMD.replicationInfo.status;
     }
     if (objectMD.replicationInfo &&
+        objectMD.replicationInfo.backends) {
+        const statuses = objectMD.replicationInfo.backends.map(backend => {
+            const key = `x-amz-${backend.site}-replication-status`;
+            responseMetaHeaders[key] = backend.status;
+            return backend.status;
+        });
+        responseMetaHeaders['x-amz-replication-status'] =
+            getReplicationStatus(statuses);
+    }
+    if (objectMD.replicationInfo &&
+        // Use storageType to determine if user metadata is needed.
         objectMD.replicationInfo.storageType &&
         objectMD.replicationInfo.status === 'COMPLETED') {
         const { storageType, dataStoreVersionId } = objectMD.replicationInfo;
```

Comment:
To follow: we will also need multiple `dataStoreVersionId`s that map to each external backend replica object.